django/test/client.py

```diff
@@ -601,12 +601,9 @@ def login(self, **credentials):
             return False
 
     def force_login(self, user, backend=None):
-        if backend is None:
```

Comment:
Please create a Trac ticket. At first glance it seems like it should be fixed. If not, it should at least be documented as a backwards-incompatibility.